health/main.py

```diff
@@ -121,17 +119,19 @@ def job():
                      % (time.time() - started_at))
 
 
-def main():
+def main(argv=sys.argv[1:]):
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-c', '--config-path', type=argparse.FileType('r'),
+                        default=DEFAULT_CONF_PATH)
+    parser.add_argument('-d', '--debug', action='store_const',
+                        const=logging.DEBUG, default=logging.INFO)
+    args = parser.parse_args(argv)
+    logging.basicConfig(format=LOGGING_FORMAT, level=args.debug)
+
     global CONF
     try:
-        with open(CONF_PATH) as f:
-            CONF = json.load(f)
-            jsonschema.validate(CONF, CONF_SCHEMA)
-
-    except (OSError, IOError):
```

Comment:
I don't like this change, without it debugging is hard. If we would like to refactor this we can change print() to logs